src/agent/main.cpp

```diff
@@ -283,7 +283,8 @@ static int realmain(int argc, char *argv[])
     }
 
     {
-        otbr::Ncp::ControllerOpenThread ncpOpenThread{interfaceName, radioUrls, backboneInterfaceName};
+        otbr::Ncp::ControllerOpenThread ncpOpenThread{interfaceName, radioUrls, backboneInterfaceName,
+                                                      printRadioVersion /* DryRun */};
```

Comment:
nit: we typically apply the comment before
```suggestion
                                                      /* aDryRun */ printRadioVersion};
```